test/Polly.Specs/CircuitBreaker/AdvancedCircuitBreakerAsyncSpecs.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Scenario = Polly.Specs.Helpers.PolicyExtensionsAsync.ExceptionAndOrCancellationScenario;
+﻿using static Polly.Specs.DictionaryHelpers;
```

Comment:
There's still a bunch of these that can be removed - otherwise this looks much neater 👍